dspy/evaluate/evaluate.py

```diff
@@ -29,17 +34,40 @@ def HTML(x):
 
 
 class Evaluate:
+    """
+    Reification of the process of evaluating a model.  Invoked using its
+    `__call__` method.
+
+    Parameters
+    ==========
+    devset: Iterable[Example]
+       An iterable of Examples
+    metric: Callable, optional
+       A function that takes an Example and a Prediction and keyword args as arguments
+       and returns a boolean
+    num_threads: int, optional
+       How many
+    display_progress: bool, optional
+    display_table: Union[bool, int], optional
+        This is either a boolean indicating whether or not to display a table of
+        results or an integer indicating how many rows of results to display.
+        If `True`, all rows will be displayed.
+    max_errors: int, optional
+    return_outputs: bool, optional
+        Whether or not to return the predictions.
```

Comment:
return_outputs is for returning the evaluations of the program.

Reply:
Thanks again.  Fixed.